frontend/src/concepts/modelCatalog/useModelCatalogSources.ts

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { getConfigMap } from '~/api';
+import useNamespaces from '~/pages/notebookController/useNamespaces';
+import { useIsAreaAvailable } from '~/concepts/areas';
+import { SupportedArea } from '~/concepts/areas/types';
+import useFetchState, { NotReadyError, FetchState } from '~/utilities/useFetchState';
+import { ModelCatalogSource } from './types';
+import { MODEL_CATALOG_SOURCE_CONFIGMAP } from './const';
+
+type State = ModelCatalogSource[];
+
+// Temporary implementation for MVP - will be replaced with API for remote model catalog sources
+// See: https://issues.redhat.com/browse/RHOAISTRAT-455
+export const useModelCatalogSources = (): FetchState<State> => {
+  const { dashboardNamespace } = useNamespaces();
+  const isModelCatalogAvailable = useIsAreaAvailable(SupportedArea.MODEL_CATALOG).status;
+
+  const callback = React.useCallback(async () => {
+    if (!isModelCatalogAvailable) {
+      throw new NotReadyError('Model catalog feature is not enabled');
+    }
+
+    const configMap = await getConfigMap(dashboardNamespace, MODEL_CATALOG_SOURCE_CONFIGMAP);
+
+    if (!configMap.data?.modelCatalogSource) {
+      return [];
+    }
+
+    const source: ModelCatalogSource = JSON.parse(configMap.data.modelCatalogSource);
+    return [source];
```

Comment:
JSON.parse can throw an exception, and we don't want invalid JSON in the configmap to crash the UI. You'll need to wrap these 2 lines in a `try { } catch(e) { }`, and in the catch block just return `[]`. I think it's ok to swallow errors like this for our temporary summit implementation.